ament_cmake_python/CMakeLists.txt

```diff
@@ -4,31 +4,6 @@ project(ament_cmake_python NONE)
 
 find_package(ament_cmake_core REQUIRED)
 
-set(ament_cmake_python_DIR "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
```

Comment:
@hidmic I think we should've deleted the [test](https://github.com/ament/ament_cmake/tree/master/ament_cmake_python/test) folder, right?

Reply:
Yeap. I'll send a PR.